src/PermissionManager.sol

```diff
@@ -20,33 +20,38 @@ import {UserOperation, UserOperationLib} from "./utils/UserOperationLib.sol";
 /// @author Coinbase (https://github.com/coinbase/smart-wallet-permissions)
 contract PermissionManager is IERC1271, Ownable2Step, Pausable {
     /// @notice Authentication data for signature validation over user operation hashes.
+    /// NOTE: I find AuthData kinda a confusing name. Should it be like CallSpecification or something? 
+    /// wasn't immediately obvious to me we needed this per call. 
     struct AuthData {
         /// @dev User operation (v0.6) to validate for.
         UserOperation userOp;
-        /// @dev Signature over user operation from permission signer.
+        /// @dev `Permission.signer` signature of user operation hash.
         bytes userOpSignature;
-        /// @dev Cosignature over user operation from cosigner.
+        /// @dev `this.cosigner` signature of user operation hash.
         bytes userOpCosignature;
-        /// @dev Permission details, approved by user i.e. smart wallet
+        /// @dev Permission details
         Permission permission;
     }
 
-    /// @notice A time-bound permission over an account given to an external signer.
+    /// @notice A limited permission for an external signer to use an account.
     struct Permission {
-        /// @dev Smart wallet address this permission is valid for.
+        /// @dev smart account address this permission is valid for.
         address account;
         /// @dev Chain this permision is valid for.
         uint256 chainId;
-        /// @dev Unix timestamp this permission is valid until.
+        /// @dev Unix timestamp in seconds this permission is valid until.
         uint48 expiry;
-        /// @dev Non-account entity given permission to sign user operations.
-        /// @dev Supports Ethereum addresses (EOA, smart contract) and P256 public keys (passkey, cryptokey).
+        /// @dev The entity that has limited control of `account` in this Permission. 
+        /// @dev Supports abi-encoded Ethereum addresses (EOA, smart contract) and P256 public keys (passkey, cryptokey).
         bytes signer;
         /// @dev External contract to verify specific permission logic.
+        /// NOTE: Maybe permissionValidator or something? 
         address permissionContract;
         /// @dev Permission-specific values sent to permissionContract for validation.
+        /// NOTE: Should this be permissionData ? 
         bytes permissionValues;
         /// @dev Manager contract that verifies permissions for replay protection across potential future managers.
+        /// NOTE: hmmm not sure i understand this. 
         address verifyingContract;
```

Comment:
You're right that we could just ignore this in the struct. I was thinking having it helps our backend services keep track of what exact manager was signed over in the case we have multiple. Less of an immediate need and more of trying to prevent a headache migrating data models in our indexing service retroactively. I'd be down to throw this out on the grounds of calldata minimization as well after trying to squeeze out more performance.